src/App.tsx

```diff
@@ -1,14 +1,58 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './utilites/todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [filtered, setFiltered] = useState<Todo[]>(todos);
+  const [loaded, setLoaded] = useState(false);
+  const [selected, setSelected] = useState<Todo | null>(null);
+  const [state, setState] = useState('All');
+
+  useEffect(() => {
+    setLoaded(true);
+
+    setTimeout(() => {
+      getTodos().then(todosFromServer => {
+        setTodos(todosFromServer);
+      }).finally(() => setLoaded(false));
+    }, 1000);
```

Comment:
Looks like you forgot to remove testing `setTimeout`
```suggestion
    getTodos().then(todosFromServer => {
      setTodos(todosFromServer);
    }).finally(() => setLoaded(false));
```